curr_scraper/apps/api_caller/management/commands/scrape.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand
+from api_caller.models import RateRecord
+from dotenv import load_dotenv
+from datetime import datetime
+import requests
+import os
+import json
+import pytz
+
+load_dotenv()
```

Comment:
You should never access ENV vars directly from apps. Use settings to get configurations.